### ISSUE TITLE
Add display name to Metric Enum

### ISSUE THREAD

**Issue by dlmarion** [apache/accumulo#4988]
Add a display name that can be used by the Monitor

**Comment by Suvrat1629** [comment]
So I need to add a new `displayName` field to the Metric constructor and then update all the enum values. Is this approach right? Do I need to look into any more files?

**Comment by Suvrat1629** [comment]
@dlmarion Please could you confirm my approach from the previous comment. I am willing to work on this issue. Thank you:)

**Comment by dlmarion** [comment]
Yes, if you are not familiar with this issue, then here is some background. There  is a plugin that runs during the build (`mvn clean package` should be enough to do it) that will generate the file `core/target/generated-docs/metrics3.md`. This markdown file at some point will be copied over to the `accumulo-website` repository (see https://github.com/apache/accumulo-website/pull/442 for example). I'm proposing that we add a display name (e.g. `Server Low Memory Indicator`) instead of the metric name `accumulo.detected.low.memory` at the start of each metric. We should include the metric name in the body of each section. The plugin may need to be modified as well to use the display name as the title and to put the metric name in the body of each metric.

@DomGarguilo wrote the plugin I believe, so he should be able to answer questions about this as well.

**Comment by Amemeda** [comment]
I Looked into this and discussed with Dave, it looks like this has been done at some point, all the tables in the Servers dropdown (Compactos, GC, Managers..) currently have a hover text description for every column except for Server Address and Resource Group which are self explanatory. The column descriptions in the Monitor UI are using the text from Metric.java, and only the Last Contact column is using text from TableDataFactory.java.

**Comment by dlmarion** [comment]
Closing this issue as the work has been done already.

**Comment by dlmarion** [comment]
Work was completed as part of #6278
